README.md

```diff
@@ -1,4 +1,4 @@
-[![Code Climate](https://codeclimate.com/repos/54242d7ce30ba04b220066b1/badges/32deaad52cf116f360f3/gpa.svg)](https://codeclimate.com/repos/54242d7ce30ba04b220066b1/feed)[![Test Coverage](https://codeclimate.com/repos/54242d7ce30ba04b220066b1/badges/32deaad52cf116f360f3/coverage.svg)](https://codeclimate.com/repos/54242d7ce30ba04b220066b1/coverage)[![Gem Version](https://badge.fury.io/rb/cb-api.svg)](http://badge.fury.io/rb/cb-api)
+[![Code Climate](https://codeclimate.com/repos/54242d7ce30ba04b220066b1/badges/32deaad52cf116f360f3/gpa.svg)](https://codeclimate.com/repos/54242d7ce30ba04b220066b1/feed)[![Test Coverage](https://codeclimate.com/repos/54242d7ce30ba04b220066b1/badges/32deaad52cf116f360f3/coverage.svg)](https://codeclimate.com/repos/54242d7ce30ba04b220066b1/coverage)[![Gem Version](https://badge.fury.io/rb/cb-api.svg)](http://badge.fury.io/rb/cb-api) [![Build Status](https://travis-ci.org/careerbuilder/ruby-cb-api.svg)](https://travis-ci.org/careerbuilder/ruby-cb-api)
```

Comment:
Some spacing in this would be nice.
![image](https://cloud.githubusercontent.com/assets/1957170/16593686/967e6128-42b4-11e6-8cdf-c2588a491285.png)